components/03-modules/gallery/Gallery.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div :class="['gallery', horizontal ? 'gallery--horizontal' : 'gallery--vertical']">
+    <div :class="['gallery__stage', horizontal ? 'gallery__stage--horizontal' : 'gallery__stage--vertical']">
+      <div class="gallery__frame">
+        <alpaca-badge
+          v-if="productLabel"
+          :class="productLabel.class"
+        >
+          {{ productLabel.text }}
+        </alpaca-badge>
+        <alpaca-image
+          :src="imageMain.dataSrc"
+          :alt="imageMain.alt"
+        />
+      </div>
+    </div>
+    <div :class="['gallery__nav', horizontal ? 'gallery__nav--horizontal' : 'gallery__nav--vertical']">
+      <div class="gallery__thumb-arr">
+        <alpaca-icon
+          icon="angle-up"
+          :class="['gallery__icon-arrow', horizontal ? 'gallery__icon-arrow--horizontal' : 'gallery__icon-arrow--vertical']"
+        />
+      </div>
+      <div
+        v-for="(thumb, i) in thumbs"
+        :key="i"
```

Comment:
It's not a good practice to use an item index as keys, because if you remove/move one element, all other items will get a new key (b/c index was changed) and will be re-rendered, even if nothing was changed.

https://vuejs.org/v2/guide/list.html#key
https://forum.vuejs.org/t/v-for-with-simple-arrays-what-key-to-use/13692/11